python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -707,10 +706,8 @@ def __init__(self, *, config=None):
             self.outputs.remove("direct")
         if not config.makePsfMatched:
             self.outputs.remove("psfMatched")
-        # TODO DM-28769: add connection per selectImages connections
-        # instead of removing if not PsfWcsSelectImagesTask here:
         if config.select.target != lsst.pipe.tasks.selectImages.PsfWcsSelectImagesTask:
-            self.inputs.remove("srcList")
+            self.inputs.remove("visitSummary")
```

Comment:
So the comment that you removed implies to me that this connection should actually be added to the `PsfWcsSelectImagesTask` itself?

Reply:
You're right, I should not have removed that to-do.